drivers.c

```diff
@@ -65,13 +65,8 @@ static char *_sub_meta_values(char *str)
 		_nt = _nt + (_at - (str + _end));
 		_var = strchr(_t, '}');
 		_end = _var - (str);
-		_start = _at - (str + _start);
```

Comment:
Yes, is used, but always it will be replaced with the value of `_end + 1` at the end of the loop and in between of those two assignments is never used

Reply:
true